code/modules/awaymissions/corpse.dm

```diff
@@ -216,6 +216,25 @@
 	qdel(src)
 
 
+/obj/effect/mob_spawn/mouse
+	name = "sleeper"
```

Comment:
should these not have special names or anything? Or is the joke that players spawn as a cow